### PR TITLE
Add cp3.13 wheel build

### PR DESCRIPTION
**Pull request by mtsokol**
This PR adds Python 3.13 wheel build.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,9 +1,13 @@
 name: Wheel
 
 on:
-  schedule:
+  pull_request:
+    branches:
+      - main  # NOTE: to remove
+  push:
+    branches:
+      - main  # NOTE: to remove
```
Comment (mtsokol) [inline .github/workflows/ci.yml:9]: I will revert it before merging.

File: pyproject.toml [hunk 2]
```diff
@@ -11,17 +11,12 @@ requires = [
...
-    "scipy",
-    "sparse@git+https://github.com/pydata/sparse@updated-llvm-nightly-test"
-]
+test-requires = ["pytest", "pytest-cov", "PyYAML", "scipy"]
+before-test = "pip install --no-deps sparse@git+https://github.com/pydata/sparse@updated-llvm-nightly-test"
 test-command = "SPARSE_BACKEND=MLIR pytest --pyargs sparse.mlir_backend"
 
 [tool.cibuildwheel.linux]
-build = "cp310-* cp311-* cp312-*"
+build = "cp310-* cp311-* cp312-* cp313-*"
```
Comment (hameerabbasi) [inline pyproject.toml:19]: And this one? Have just one in `tool.cibuildwheel`.

**Review by hameerabbasi — CHANGES_REQUESTED** [review]
A couple of cleanup questions.

File: pyproject.toml [hunk 3]
```diff
@@ -31,7 +26,7 @@ before-build = [
 environment-pass = ["HOST_CCACHE_DIR"]
 
 [tool.cibuildwheel.macos]
-build = "cp310-* cp311-* cp312-*"
+build = "cp310-* cp311-* cp312-* cp313-*"
```
[the remark this answers is not in the document]
Comment (mtsokol) [reply pyproject.toml]: Done!

File: pyproject.toml [hunk 4]
```diff
@@ -41,7 +36,7 @@ repair-wheel-command = [
 ]
 
 [tool.cibuildwheel.windows]
-build = "cp310-* cp311-* cp312-*"
+build = "cp310-* cp311-* cp312-* cp313-*"
```
[the remark this answers is not in the document]
Comment (mtsokol) [reply pyproject.toml]: Done!

File: pyproject.toml (hunk 2 above)
Comment (mtsokol) [reply pyproject.toml]: Done!

**Review by hameerabbasi — APPROVED** [review]

**Comment by hameerabbasi** [comment]
Thanks for working on this @mtsokol!

**Pull request merged**